docs/pages/database-access/architecture.mdx

```diff
@@ -1,13 +1,14 @@
 ---
-title: Database Access Architecture
-description: How Teleport Database Access works.
+title: Database Service Architecture
+description: How Teleport the Database Service works.
```

Comment:
This page isn't describing the architecture of just the database service, it's describing the architecture of the entire database access feature.

If you don't want to say _Database Access_ anymore, that's fine, but I think we should find a title that makes it clear this  is about how Teleport provides access to databases, which involves not only the database service, but the auth service and proxy service.